config/defines.inc.php

```diff
@@ -200,7 +200,7 @@
 define('ALL_CARRIERS', 5);
 
 /* SQL Replication management */
-define('_PS_USE_SQL_SLAVE_', 0);
+define('_PS_USE_SQL_SLAVE_', false);
```

Comment:
Mmm, isn't it a BC I'm sure there are places (in core or modules) that could use if `0 === _PS_USE_SQL_SLAVE_ `

Reply:
Correct. It's a wanted BC, discussed with @PierreRambaud. I document it.

Reply:
Documented.